templates/projects/_helpers.html

```diff
@@ -56,6 +56,10 @@
     {% if is_subadmin_and_owner_or_admin %}
         <a class="list-group-item {% if active_link=='auditlog'%}active{% endif %}" href="{{url_for('project.auditlog', short_name=project.short_name)}}">{{_('Audit Logs')}}<i class="fa fa-chevron-right pull-right"></i></a>
     {% endif %}
+
+    {% if is_admin_or_subadmin_or_owner %}
```

Comment:
as project creation access is to admin / subadm, I think this would be is_subadmin_and_owner_or_admin instead. this would also be inline with backend.